app-accessibility/yasr/yasr-0.6.9-r1.ebuild

```diff
@@ -39,11 +51,6 @@ src_configure() {
 src_install() {
```

Comment:
Remove src_install. Default behaviour is same.
https://devmanual.gentoo.org/ebuild-writing/functions/src_install/index.html

Reply:
Remove src_install step. It isn't needed.